nrcemt/qeels/gui/main_window.py

```diff
@@ -300,15 +332,19 @@ def detect(self):
         for item in self.width_array:
             width.append(item.width_var.get())
             checkbox.append(item.detect_var.get())
+
+        index = self.list.curselection()
```

Comment:
give index a more specific name, or move the `self.list.curselection()` call onto the next line